source/elements/oneMKL/source/domains/blas/gemm_batch.rst

```diff
@@ -461,6 +465,181 @@ in ``a``, ``b`` and ``c`` are given by the ``batch_size`` parameter.
 
    Output event to wait on to ensure computation is complete.
 
+**Span API**
+
+.. rubric:: Description
+
+.. rubric:: Syntax
+
+.. code-block:: cpp
+
+   namespace oneapi::mkl::blas::column_major {
```

Comment:
Do we need to have two different full version for span and pointer here?
Since they are overload can't we do like for the data type use a T_array (and other as needed) type and then list what T_array are supported?
Maybe it would require too many different types and it is not a good idea but that would be good to give it a try and see how it could look.
This would allow to avoid all the copy-paste for the input and output explanation. We would have one description only of the group API inputs and outputs handling both cases pointer and span inputs.